.github/workflows/ci-integration-db.yaml

```diff
@@ -0,0 +1,58 @@
+name: CI - DB Integration
+
+on: [push, pull_request]
+
+jobs:
+  database-integration-test:
+    strategy:
+      matrix:
+        python-version: ['3.11']
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    services:
+    # postgreSQL Service
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: db
+          POSTGRES_USER: ploomber_app
+          POSTGRES_PASSWORD: ploomber_app_password
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+      # mySQL Service
+      mysql:
+        image: mysql:5.7
```

Comment:
looks like 8 is the latest version, what happens if we put `image: mysql`? I'm assuming it'll get us the latest version

Reply:
Addressed in https://github.com/ploomber/jupysql/pull/111/commits/a7e8a44932a2d5e2c415bcf55c1523951b1c609d

Seems latest mysql is fine, we'll make a dynamic MySQL version here in the next